skyportal/handlers/api/comment.py

```diff
@@ -748,17 +748,17 @@ def put(self, associated_resource_type, resource_id, comment_id):
             elif isinstance(c, CommentOnGCN):  # also update the gcn
                 self.push_all(
                     action='skyportal/REFRESH_SOURCE_GCN',
-                    payload={'obj_internal_key': c.obj.internal_key},
+                    payload={'gcnEvent_dateobs': c.gcn.dateobs},
                 )
             elif isinstance(c, CommentOnEarthquake):  # also update the earthquake
                 self.push_all(
                     action='skyportal/REFRESH_EARTHQUAKE',
-                    payload={'obj_internal_key': c.obj.internal_key},
+                    payload={'earthquake_id': c.earthquake.event_id},
                 )
             elif isinstance(c, CommentOnShift):  # also update the shift
                 self.push_all(
                     action='skyportal/REFRESH_SHIFT',
-                    payload={'obj_internal_key': c.obj.internal_key},
+                    payload={'shift_id': c.shift_id},
```

Comment:
How did this even work until now??

Reply:
well we dont have any front-end mechanism for editing comments...